core/src/main/java/com/novoda/noplayer/PlayerSubtitlesView.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.noplayer;
+
+import com.google.android.exoplayer2.text.Cue;
+
+import java.util.List;
+
+public interface PlayerSubtitlesView {
```

Comment:
Otherwise I think this is still ok, but maybe with a renaming, `PlayerSubtitlesView` sounds weird to me (maybe just me)